pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -48,6 +48,10 @@ let
       "-DSLIC3R_BUILD_TESTS=0"
     ];
 
+    buildInputs = super.buildInputs ++ [
+      libspnav
```

Comment:
wow Merill added spacemouse support?

Reply:
looks like it :) sadly it doesn't compile due to boost issues. I hope I find a working patch.